### PR TITLE
Update the single triggers plot to use glob rather than bash wildcards

### PR DESCRIPTION
**Pull request by GarethCabournDavies**
EW analysis make many many more files than the full bandwidth version, so we need to update to use glob which can handle more files than adding as arguments

### CODE REVIEW

**Comment by GarethCabournDavies** [comment]
I dont see how to ask for reviewers, but will tag @titodalcanton

**Comment by GarethCabournDavies** [comment]
Test plot out at https://ldas-jobs.ligo.caltech.edu/~gareth.cabourndavies/pycbclive/singles_plot_ew_arglist/pycbc-live-utils/test.png

File: pycbclive_plot_singles.py [hunk 1]
```diff
@@ -53,12 +54,13 @@ def plot_gate(ax, gate):
 
 parser = argparse.ArgumentParser(description=__doc__)
 parser.add_argument(
-    '--trigger-files',
+    '--trigger-files-glob',
```
Comment (spxiwh) [inline pycbclive_plot_singles.py:57]: Why not use trigger-file-dir, and then avoid issues of escaping wildcards? ... If the idea is always that we just read everything in one dir? (Could always provide more than one dir).

Comment (GarethCabournDavies) [reply pycbclive_plot_singles.py:57]: That is what I was thinking - we could use something like `/path/to/dir/\*/H1L1V1-Triggers-12345\*.hdf`, where `12345*` would cross the boundaries of the directories

Comment (titodalcanton) [reply pycbclive_plot_singles.py:57]: We do not necessarily want to plot *every* file in one dir. Wildcards could be useful.

**Review by titodalcanton — APPROVED** [review]

**Pull request merged**